### ISSUE TITLE
Icons and text in code have different font sizes

### ISSUE THREAD

**Issue by brian-swan** [Microsoft/TouchDevelop#59]
The icons used in code (e.g. the flower for art, the arrows for library, etc.) have a different font size that the rest of the text used in writing code. This causes line spacing to vary, depending on whether a line contains an icon. This can make code more difficult to read.

**Comment by pelikhan** [comment]
Could you give a screenshot that shows the problem?

**Comment by brian-swan** [comment]
Sorry. I filed this based on feedback from Sublime. After clarification with them, there is no issue. Closing.

**Comment by pelikhan** [comment]
I think it would be good to talk about those icons in beginner mode. IMO, we should use the full name in that mode. Beginners get confused and have issues finding it.

Sent from Outlook<http://taps.io/outlookmobile>




On Fri, Feb 27, 2015 at 11:58 AM -0800, "Brian Swan" <notifications@github.com<mailto:notifications@github.com>> wrote:


Sorry. I filed this based on feedback from Sublime. After clarification with them, there is no issue. Closing.

-
